syntax/std_json_test.go

```diff
@@ -0,0 +1,42 @@
+package syntax
+
+import "testing"
+
+func TestJSONDecode(t *testing.T) {
+	t.Parallel()
+	AssertCodePanics(t, `//.json.decode(123)`)
+	AssertCodesEvalToSameValue(t,
+		`{
+			"a": (s: "string"),
+			"b": 123,
+			"c": 123.321,
+			"d": (a: [1, (s: "string again"), (a: []), {}]),
+			"e": {
+				"f": {
+					"g": (s: "321")
+				},
+				"h": (a: [])
+			},
+			"i": (null: {}),
+			"j": (a: [(b: {()}), (b: {})]),
```

Comment:
You can write `true` instead of `{()}` and `false` instead of `{}`. They are exactly the same thing, but it conveys the intent better.